builder/package.rhel-9

```diff
@@ -0,0 +1,76 @@
+if [[ ! -d /tmp/output/rhel-9 ]]; then
+  mkdir -p /tmp/output/rhel-9
+fi
+
+# R 3.x requires PCRE1
+pcre_lib='pcre2-devel'
+if [[ "${R_VERSION}" =~ ^3 ]]; then
+  pcre_lib='pcre-devel'
+fi
+
+# Create post-install script required for OpenBLAS.
+#
+# On RHEL and SUSE, we link R against the internal shared BLAS to make the
+# R binaries more portable and allow users to switch BLAS implementations without
+# having to recompile R. We default to OpenBLAS, but users may prefer other implementations.
+# 
+# Binary packages built against the shared BLAS are also more portable and may be used
+# with the default R distributed by RHEL/SUSE, or other R installations using
+# shared BLAS and configured with a different BLAS (such as Microsoft R Open with MKL).
+# This is especially important for RSPM's binary packages.
+#
+# On Ubuntu/Debian, we link R against the external BLAS instead (--with-blas/--with-lapack),
+# as those distributions use the alternatives system to swap BLAS libraries at runtime.
+# The default R distributions on Ubuntu/Debian use the external BLAS, so we do as well
+# for portability.
+#
+# https://cran.r-project.org/doc/manuals/r-release/R-admin.html#Shared-BLAS
```

Comment:
@michaelmayer2 I've added some docs to explain how we configure BLAS here (https://github.com/rstudio/r-builds/pull/132)

I'll add a similar comment to the openSUSE 15.4 build as well. I didn't copy these over to the previous CentOS versions since we usually just add new platforms based off the most recent version of that distro.